### DISCUSSION TITLE
Web3Modal within an iFrame not calling to open Mobile Wallet (Safari & Chrome)

### DISCUSSION THREAD

**Discussion by 0x4Graham** [WalletConnect/walletconnect-monorepo#4313] [Developer Support]
Hello, 

I've currently implemented Web3Modal into my app. However, my app lives within an iFrame. 

Currently, Web3Modal works perfectly fine on Desktop. However, on a mobile device, such as an iPhone using Safari. The call to open the mobile app doesn't make the call to open the mobile wallet. See screenshot attached: 
![IMG_41FF2CA60DF4-1](https://github.com/WalletConnect/walletconnect-monorepo/assets/7548837/697cd3dd-e644-42b6-ad73-2fdb1921a556)

I'm getting these console errors via the safari browser on mobile and not sure they will impact? 

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/7548837/526f0265-63eb-4596-960f-55677fcb6ae6)

    "viem": "^2.7.20",
    "wagmi": "^2.5.7",
    "@wagmi/core": "^2.6.5",
    "@web3modal/wagmi": "^4.0.13",

**Comment by 0x4Graham** [comment]
Interesting that Coinbase wallet works, but not Zerion or Rainbow

**Reply by glitch-txs** [reply]
could you share a minimal reproducible example & steps to reproduce? Thanks!

**Comment by CaioRutka** [comment]
Man, i cant even make the WalletCoonect to appear on screen on mobile browser..
